### PR TITLE
Add persistence for reporting.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: src/main/java/io/snowdrop/github/reporting/GithubReportingService.java [hunk 1]
```diff
@@ -1,19 +1,92 @@
...
 public class GithubReportingService {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(GithubReportingService.class);
+
   @Inject
   GithubReporting reporting;
 
+  @ActivateRequestContext
+  void onStart(@Observes StartupEvent ev) {
+    popullateRepos();
```
Comment (geoand) [inline src/main/java/io/snowdrop/github/reporting/GithubReportingService.java:31]: `popullateRepos` is spelled incorrectly, should be `populateRepos`

File: src/main/java/io/snowdrop/github/reporting/GithubReportingService.java [hunk 2]
```diff
@@ -1,19 +1,92 @@
...
 
 @Singleton
 public class GithubReportingService {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(GithubReportingService.class);
+
   @Inject
   GithubReporting reporting;
 
+  @ActivateRequestContext
```
Comment (geoand) [inline src/main/java/io/snowdrop/github/reporting/GithubReportingService.java:29]: Remind please why this is needed?

File: src/main/java/io/snowdrop/github/reporting/model/IssueDTO.java [hunk 3]
```diff
@@ -2,10 +2,16 @@
...
 
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
 import org.eclipse.egit.github.core.Issue;
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
 
-public class IssueDTO {
+@Entity
+public class IssueDTO extends PanacheEntityBase {
```
Comment (geoand) [inline src/main/java/io/snowdrop/github/reporting/model/IssueDTO.java]: It's kind of weird to name an entity `DTO`, isn't?

File: src/main/java/io/snowdrop/github/reporting/GithubReportingService.java (hunk 2 above)
Comment (iocanel) [reply src/main/java/io/snowdrop/github/reporting/GithubReportingService.java:29]: I was having issues when trying to do jpa stuff from `onStart`. 
After searching for a while, I bumped to post suggesting adding the annotation.

Comment (geoand) [reply src/main/java/io/snowdrop/github/reporting/GithubReportingService.java:29]: OK, let's keep it

File: src/main/java/io/snowdrop/github/reporting/model/IssueDTO.java (hunk 3 above)
Comment (iocanel) [reply src/main/java/io/snowdrop/github/reporting/model/IssueDTO.java]: Well, it was a DTO before becoming an entity ;-)

Let me see, if I can waste the suffix without clashing with egit types.

Comment (iocanel) [reply src/main/java/io/snowdrop/github/reporting/model/IssueDTO.java]: @geoand renamed!

**Review by geoand — APPROVED** [review]

**Pull request merged**